src/types/result.ts

```diff
@@ -0,0 +1,20 @@
+
+export type ResultData = (
+  flagSet?: string,
+  overrides?: { [flagId: string]: { heading?: string; description?: string } }
```

Comment:
nit: Could we extract this interface out and export it so that `getResultData` can refer to the same type directly?

Reply:
Yep, done!